examples/python-sdk-demo/main.py

```
@@ -0,0 +1,267 @@
+        )
+
+        print(f"  Uploading '{test_file_path.name}'...")
+        with open(test_file_path, "rb") as f:
+            result = client.storage.upload(
+                f,
+                filename=test_file_path.name,
+                token=token,
+            )
+        print(f"  ✅ Upload successful! Path: {result.get('path', 'N/A')}")
+
+        print(f"\n  Getting download URL...")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)